lib/src/main/java/com/google/android/material/composethemeadapter/TypedArrayUtils.kt

```diff
@@ -215,7 +215,7 @@ internal fun TypedArray.getFontFamilyOrNull(index: Int): FontFamilyWithWeight? {
             else -> {
                 // If there's a resource ID and the string starts with res/font,
                 // it's probably a @font resource
-                if (tv.resourceId != 0 && tv.string.startsWith("res/font")) {
+                if (tv.resourceId != 0 && tv.string.startsWith("res/")) {
```

Comment:
This path check is now pretty useless, we can probably remove the `&& tv.string.startsWith("res/")`

Reply:
For now this check still has a purpose.
We could have set the font family by a string resource. If this string is one of the not yet handled ones (https://github.com/material-components/material-components-android-compose-theme-adapter/blob/develop/lib/src/main/java/com/google/android/material/composethemeadapter/TypedArrayUtils.kt#L214) than we would instanciate a `Font` with a string resource id instead of an font resource id.

Reply:
Ah yes 👍